docs/docs/guides/getting_started.md

```diff
@@ -5,9 +5,17 @@ title: Getting started
 
 # Getting Started
 
+:::tip
+
+If you already have an environment with Linux and Docker installed, you can continue to [Installing Frigate](#installing-frigate) below.
+
+If you already have Frigate installed, you can continue to [Configuring Frigate](#configuring-frigate) below.
```

Comment:
```suggestion
If you already have Frigate installed as a Home Assistant addon, you can continue to [Configuring Frigate](#configuring-frigate) below.
```

Reply:
For this one I think it might make sense to leave generic (but perhaps it can be reworded to make more sense)? I am imagining it would be useful for any user who might have followed the installation docs to have a quick link to configuring Frigate